programs/programs/cross_white_label/medicaid/tests/test_wa.py

```
@@ -0,0 +1,325 @@
+    # ------------------------------------------------------------------
+
+    def test_premium_chip_ceiling_hh3(self):
+        """HH3 ceiling matches the hardcoded HCA 2026 value."""
+        self.assertEqual(WaAppleHealthMedicaid._premium_chip_ceiling(3), 86_604)
+
+    def test_premium_chip_ceiling_hh1(self):
+        """HH1 ceiling matches the hardcoded HCA 2026 value."""
+        self.assertEqual(WaAppleHealthMedicaid._premium_chip_ceiling(1), 50_591)
+
+    def test_premium_chip_ceiling_hh8_extrapolated(self):
+        """HH8 is extrapolated: HH7 base + 1 × additional-member increment."""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)